ivy_tests/test_ivy/test_frontends/test_tensorflow/test_signal.py

```diff
@@ -0,0 +1,59 @@
+import ivy
+
+from hypothesis import given, assume, strategies as st
+from ivy.functional.backends.numpy.data_type import dtype
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+# hann_window
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric")
+    ),
+    # generate pereodic boolean
+    peroidic=st.booleans(),
+    dtype=helpers.get_dtypes("float", full=False, none=True),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.signal.hann_window"
+    ),
+)
+def test_tensorflow_hann_window(
+    dtype_and_x,
+    dtype,
+    periodic,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, x = dtype_and_x
+
+    # test if periodic is a boolean
+    if periodic is not None:
+        assume(isinstance(periodic, bool))
+
+    as_variable, native_array = helpers.handle_where_and_array_bools(
```

Comment:
as there is no where argument in hann_window, this part of using handle_where_and_array_bools should not be added.